app/controllers/api/v1/review_controller.rb

```diff
@@ -32,7 +32,7 @@ def create
 
         # current 
         @review = Review.new(text: params[:value], business_id: @business.id, user_id: session[:user_id])
-        puts @review.user.username, "ohhh loard please"
+        # puts @review.user.username, "ohhh loard please"
```

Comment:
Probably want to just delete this line 😄

Reply:
Deleted it